### PR TITLE
setup the application for localstack usage

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,119 +1,98 @@
-# Friend Microservices Sample
+# Serverless microservices with Amazon API Gateway, Amazon DynamoDB, Amazon SQS, and AWS Lambda
```
Comment (whummer) [inline README.md]:
```suggestion
# Serverless Microservices with Amazon API Gateway, DynamoDB, SQS, and Lambda
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION=us-east-1
+SHELL := /bin/bash
+
+## Show this help
+usage:
+		@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
```
Comment (whummer) [inline Makefile]: nit: looks like we're using 2 tabs indentation in here, could reduce the level to 1 tab only

File: README.md [hunk 3]
```diff
@@ -1,119 +1,98 @@
...
-![alt text](./docs/FriendsMicroservices.png)
+TODO
 
-## DynamoDB Tables
+## Prerequisites
 
-### Friend Table
+- LocalStack Pro
+- [AWS CLI](https://docs.localstack.cloud/user-guide/integrations/aws-cli/) with the `awslocal` wrapper.
+- [CDK](https://docs.localstack.cloud/user-guide/integrations/aws-cdk/) with the `cdklocal` wrapper.
```
Comment (whummer) [inline README.md]:
```suggestion
- [CDK](https://docs.localstack.cloud/user-guide/integrations/aws-cdk/) with the [`cdklocal`](https://github.com/localstack/aws-cdk-local) wrapper.
```

File: README.md [hunk 4]
```diff
@@ -1,119 +1,98 @@
...
-There also is an API Gateway and a Lambda Function for reading data\
-![alt text](./docs/FriendsMicroservices.png)
+TODO
 
-## DynamoDB Tables
+## Prerequisites
 
-### Friend Table
+- LocalStack Pro
+- [AWS CLI](https://docs.localstack.cloud/user-guide/integrations/aws-cli/) with the `awslocal` wrapper.
```
Comment (whummer) [inline README.md]:
```suggestion
- [AWS CLI](https://docs.localstack.cloud/user-guide/integrations/aws-cli/) with the [`awslocal`](https://github.com/localstack/awscli-local) wrapper.
```

**Review by whummer — APPROVED** [review]
Very cool - works like a charm (on Macbook M1 here). 🎉  

----

Only had to make a small adjustment/fix to our implementation to get this working with latest (ran into some S3 `Forbidden` errors) - will create a follow-up PR in localstack in a bit..

Update: Please ignore the comment above - should be working with the `latest` image (was testing in host mode). 👍

**Review by tinyg210 — APPROVED** [review]

**Pull request merged**